commit 707692e67d0c6c685c42f5dd48f8f112bde6b381
Author: Tomasz Zawadzki <tomasz.zawadzki@intel.com>
Date:   Thu Jan 14 04:36:56 2021 -0500

    build/meson: disable qat_asym driver in common/qat
    
    Reason to disable qat_asym is described in:
    (d0d4c32d)build/meson: disable qat_asym driver
    
    Meanwhile due to change from DPDK 20.08 to 20.11:
    (2ca75c65)common/qat: build drivers from common folder
    it is required to implement similar change to
    common/qat meson build file.
    
    Signed-off-by: Tomasz Zawadzki <tomasz.zawadzki@intel.com>
    Change-Id: Ie14350d2eeb1b680a7fe53152a1bb253261e1ebb
    Reviewed-on: https://review.spdk.io/gerrit/c/spdk/dpdk/+/5912
    Reviewed-by: Paul Luse <paul.e.luse@intel.com>
    Reviewed-by: Jim Harris <james.r.harris@intel.com>
    Reviewed-by: Shuhei Matsumoto <shuhei.matsumoto.xt@hitachi.com>

diff --git a/drivers/common/qat/meson.build b/drivers/common/qat/meson.build
index 29e1299f2..9d67cfbb2 100644
--- a/drivers/common/qat/meson.build
+++ b/drivers/common/qat/meson.build
@@ -56,10 +56,10 @@ endif
 
 if qat_crypto
 	foreach f: ['qat_sym_pmd.c', 'qat_sym.c', 'qat_sym_session.c',
-			'qat_sym_hw_dp.c', 'qat_asym_pmd.c', 'qat_asym.c']
+			'qat_sym_hw_dp.c']
 		sources += files(join_paths(qat_crypto_relpath, f))
 	endforeach
 	deps += ['security']
 	ext_deps += libcrypto
-	cflags += ['-DBUILD_QAT_SYM', '-DBUILD_QAT_ASYM']
+	cflags += ['-DBUILD_QAT_SYM']
 endif
